src/NHSD.BuyingCatalogue.Ordering.Api/Models/UpdateOrderSection.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace NHSD.BuyingCatalogue.Ordering.Api.Models
+{
+    public sealed class UpdateOrderSection
```

Comment:
For consistency the class name should end with 'Model'.

Reply:
👍